service/recognition.js

```diff
@@ -163,19 +163,19 @@ async function groupUsers(client, groupId) {
   throw new SlackError(
     "usergroups.users.list",
     response.error,
-    `Something went wrong while sending recognition. When retreiving usergroup information from Slack, the API responded with the following error: ${response.message} \n Recognition has not been sent.`
+    `Something went wrong while sending recognition. When retreiving usergroup information from Slack, the API responded with the following error: ${response.message} \n Recognition has not been sent.`,
   );
 }
 
 async function gratitudeReceiverIdsIn(client, text) {
   let users = (text.match(userRegex) || []).map((userMention) =>
-    userMention.slice(2, -1)
+    userMention.slice(2, -1),
   );
   let groups = (text.match(groupRegex) || []).map((groupMention) =>
     groupMention.substring(
       groupMention.indexOf("^") + 1,
-      groupMention.lastIndexOf("|")
-    )
+      groupMention.lastIndexOf("|"),
+    ),
```

Comment:
I had no idea (until now) that `.map` and `.filter` had an optional second argument

Reply:
Yeah, I like the formatting style with trailing commas, but I was a bit surprised to see them all over the formatter, even for functions without extra arguments. I'm not complaining though. :)